tbot_contrib/gpio.py

```diff
@@ -1,8 +1,17 @@
+import time
 from tbot.machine import linux
 
 
 class Gpio:
     def __init__(self, host: linux.LinuxShell, gpio_number: int):
+        """Initialize GPIO
+
+        Raspberry Pi Model 2 and above use as gpio_number their BCM GPIO Numbers
+
+        Args:
+            host (linux.LinuxShell):    Linux Shell
+            gpio_number (int):          GPIO Number
```

Comment:
The format for arguments is different in the sphinx configuration we're using:

```suggestion
        :param linux.LinuxShell host: Linux Shell
        :param int gpio_number: GPIO Number
```